fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -560,6 +560,72 @@ public void testUpdateObjectGraph() throws IOException {
     }
 
     @Test
+    public void testDeleteMultipleMultiValuedProperties() throws IOException {
+        final String id = getRandomUniqueId();
+        createObjectAndClose(id);
+
+        final String location = serverAddress + id;
+        final HttpPatch addTriplesGraphMethod = new HttpPatch(location);
+        addTriplesGraphMethod.addHeader(CONTENT_TYPE, "application/sparql-update");
+
+        final String insertStatement = "INSERT DATA { \n" +
+                "  <> <http://example.org/test/x> \"x\" . \n" +
+                "  <> <http://example.org/test/a> \"1\" . \n" +
+                "  <> <http://example.org/test/a> \"2\" . \n" +
+                "  <> <http://example.org/test/a> \"3\" . \n" +
+                "  <> <http://example.org/test/a> \"4\" . \n" +
+                "  <> <http://example.org/test/b> \"1\" . \n" +
+                "  <> <http://example.org/test/b> \"2\" . \n" +
+                "  <> <http://example.org/test/b> \"3\" . \n" +
+                "  <> <http://example.org/test/b> \"4\" . \n" +
+                "}";
+
+        addTriplesGraphMethod.setEntity(new StringEntity(insertStatement));
+        assertEquals(NO_CONTENT.getStatusCode(), getStatus(addTriplesGraphMethod));
+
+        // ensure that the triples are there.
+        try (final CloseableDataset dataset = getDataset(getObjMethod(id))) {
+            final DatasetGraph graph = dataset.asDatasetGraph();
+            assertTrue("Didn't find a triple we expected!", graph.contains(ANY,
+                    createURI(location), createURI("http://example.org/test/x"), createLiteral("x")));
+            assertTrue("Didn't find a triple we expected!", graph.contains(ANY,
+                    createURI(location), createURI("http://example.org/test/a"), createLiteral("1")));
+
+        }
+
+        final HttpPatch deleteQuery = new HttpPatch(location);
+        deleteQuery.addHeader(CONTENT_TYPE, "application/sparql-update");
+
+        final String deleteQueryStatement = "" +
+                "DELETE { \n" +
+                " <> <http://example.org/test/a> ?a . \n" +
+                " <> <http://example.org/test/b> ?b . \n" +
+                "} \n" +
+                "WHERE \n" +
+                "{ \n" +
+                "  <> <http://example.org/test/a> ?a . \n" +
+                "  <> <http://example.org/test/b> ?b . \n" +
+                "} ";
```

Comment:
Good point - I was going off of the script.  I'll change that up.  I think one test is sufficient since they should be be functionally equivalent, no?